index.tsx

```diff
@@ -90,11 +90,7 @@ export namespace MarkdownToJSX {
     /**
      * Ultimate control over the output of all rendered JSX.
      */
-    createElement: (
-      tag: Parameters<CreateElement>[0],
-      props: React.Props<any>,
```

Comment:
`React.Props` was flagged as deprecated, and it doesn't seem to be in the type of React.createElement anymore